docs/SAMPLES.md

```diff
@@ -28,9 +28,12 @@ const loader = await createZarrLoader(zarrInfo);
 /* Zarr loader */
 // OR
 /* Tiff Loader */
-const url =
+const urlOrFile =
   'https://vitessce-demo-data.storage.googleapis.com/test-data/deflate_no_legacy/spraggins.bioformats.raw2ometiff.ome.tif';
```

Comment:
Is there a smaller url? perhaps from the viv-demo data?

Reply:
Good call.